packages/orchestration/README.md

```diff
@@ -65,28 +65,61 @@ Consequently, each orchestration deployment uniquely maps to a resource group wi
 ## Usage
 
 Leverage the orchestration service capabilities by using the orchestration client.
-Configure the LLM module by setting the `model_name` property.
-Define the optional `model_version` property to choose an available model version.
-By default, the version is set to `latest`.
-Specify the optional `model_params` property to apply specific parameters to the model
+You can perform a simple prompt by running:
 
 ```ts
 import { OrchestrationClient } from '@sap-ai-sdk/orchestration';
 
 const orchestrationClient = new OrchestrationClient({
   llm: {
-    model_name: 'gpt-4o',
-    model_params: { max_tokens: 50, temperature: 0.1 },
-    model_version: 'latest'
+    model_name: 'gpt-4o'
   },
-  ...
+  templating: {
+    template: [
+      { role: 'user', content: 'Hello World! Why is this phrase so famous?' }
+    ]
+  }
 });
+
+const response = await orchestrationClient.chatCompletion();
+console.log(response.getContent());
 ```
 
-The client allows you to combine various modules, such as templating and content filtering, while sending chat completion requests to an orchestration-compatible generative AI model.
+Here we use GPT-4o with a single user message as prompt and print out the response.
+
+The `OrchestrationClient` allows you to combine various modules, such as templating and content filtering, while sending chat completion requests to an orchestration-compatible generative AI model.
 
 In addition to the examples below, you can find more **sample code** [here](https://github.com/SAP/ai-sdk-js/blob/main/sample-code/src/orchestration.ts).
 
+### LLM Config
+
+Configure which LLM to use module by setting the `model_name` property.
+Define the optional `model_version` property to choose an available model version.
+By default, the version is set to `latest`.
+Specify the optional `model_params` property to apply specific parameters to the model
+
+> [!Tip]
+>
+> #### Harmonized API
+>
+> As the orchestration service API is harmonized, you can switch to a different model, even from another vendor, by changing only the `model_name` property.
+> Here’s an example where only one line of code is changed.
```

Comment:
[pp] I feel like we dont need this. The above sentence say only one change is needed and we highlight that in the code

Reply:
Agreed. After reviewing this I felt that we can rather just link to the docs of orchestration where this is explained better anyway, so I shortened this to just a short explanation + link